grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -1060,23 +1061,75 @@ class QueryService implements Status {
         results
     }
 
-    List<Issue> findIssueByProjectType(String type) {
+    PaginatedResponse findIssueByProjectType(String type, PaginationParams pagination) {
+        String orderColumn = getIssueOrderColumn(pagination.orderColumn)
         SessionFactory sessionFactory = grailsApplication.getMainContext().getBean('sessionFactory')
         final session = sessionFactory.currentSession
-        final String query =
-                ' select * ' +
-                        ' from issue ' +
-                        ' where type = :projectType ' +
-                        ' and deleted = 0 '
-        final SQLQuery sqlQuery = session.createSQLQuery(query)
-        final results = sqlQuery.with {
+        final StringBuffer query = new StringBuffer(' select distinct i.id from issue i left outer join issue_extra_property ie on ie.issue_id = i.id ')
```

Comment:
It seems to me that this query can be combined with the one below in `'select * from issue where id in :ids '` and get a distinct list of all columns in the `issue` table for the criteria, and ordered.